files/etc/apt/apt.conf.d/10periodic

```diff
@@ -1,4 +1,4 @@
-APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Update-Package-Lists "0";
 APT::Periodic::Download-Upgradeable-Packages "0";
 APT::Periodic::AutocleanInterval "0";
 APT::Periodic::Unattended-Upgrade "0";
```

Comment:
Can you add new line here?

Thanks

Reply:
This file as default don't have a new line.

This new line are really needed?

Reply:
No :+1: